The ONNX.Tile5D test in tmva/sofie/test/TestCustomModelsFromONNX.cxx writes array elements beyond the last element in the array.

### Check duplicate issues.

- [X] Checked for duplicates

### Description

The test writes to random memory. The test segfaults on ix86.
```
 446/1408 Test  #410: gtest-tmva-sofie-TestCustomModelsFromONNX .....................................................***Failed   24.65 sec
Running main() from /builddir/build/BUILD/gtest-1.14.0-build/googletest-1.14.0/googletest/src/gtest_main.cc
Note: Google Test filter = -RNTuple.StdAtomic:RCsvDS.Remote:RNTuple.OpenHTTP:RRawFile.Remote:RSqliteDS.Davix:TChainParsing.DoubleSlash:TChainParsing.RemoteGlob:TFile.ReadWithoutGlobalRegistrationNet:TFile.ReadWithoutGlobalRegistrationWeb
[==========] Running 99 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 99 tests from ONNX
[ RUN      ] ONNX.Linear16
[       OK ] ONNX.Linear16 (15 ms)
[ RUN      ] ONNX.Linear32
[       OK ] ONNX.Linear32 (15 ms)
[ RUN      ] ONNX.Sub
[       OK ] ONNX.Sub (0 ms)
[ RUN      ] ONNX.Add
[       OK ] ONNX.Add (0 ms)
[ RUN      ] ONNX.Mul
[       OK ] ONNX.Mul (0 ms)
[ RUN      ] ONNX.Div
[       OK ] ONNX.Div (0 ms)
[ RUN      ] ONNX.Neg
[       OK ] ONNX.Neg (0 ms)
[ RUN      ] ONNX.Elu
[       OK ] ONNX.Elu (0 ms)
[ RUN      ] ONNX.Constant
[       OK ] ONNX.Constant (0 ms)
[ RUN      ] ONNX.ComplexTopK
54 54
Output[0]: 9, Correct[0]: 9
Output[1]: 8, Correct[1]: 8
Indexes[0]: 0, Correct[0]: 0
Indexes[1]: 0, Correct[1]: 0
[       OK ] ONNX.ComplexTopK (0 ms)
[ RUN      ] ONNX.TopK
5 5
Output[0]: 9, Correct[0]: 9
Output[1]: 8, Correct[1]: 8
Indexes[0]: 0, Correct[0]: 0
Indexes[1]: 1, Correct[1]: 1
[       OK ] ONNX.TopK (0 ms)
[ RUN      ] ONNX.EyeLike
[       OK ] ONNX.EyeLike (0 ms)
[ RUN      ] ONNX.Cast
[       OK ] ONNX.Cast (0 ms)
[ RUN      ] ONNX.Linear64
[       OK ] ONNX.Linear64 (79 ms)
[ RUN      ] ONNX.LinearWithSelu
[       OK ] ONNX.LinearWithSelu (0 ms)
[ RUN      ] ONNX.Tanh
[       OK ] ONNX.Tanh (0 ms)
[ RUN      ] ONNX.Erf
[       OK ] ONNX.Erf (0 ms)
[ RUN      ] ONNX.Log
[       OK ] ONNX.Log (0 ms)
[ RUN      ] ONNX.LinearWithLeakyRelu
[       OK ] ONNX.LinearWithLeakyRelu (0 ms)
[ RUN      ] ONNX.LinearWithSigmoid
[       OK ] ONNX.LinearWithSigmoid (0 ms)
[ RUN      ] ONNX.ConvWithPadding
[       OK ] ONNX.ConvWithPadding (0 ms)
[ RUN      ] ONNX.ConvWithoutPadding
[       OK ] ONNX.ConvWithoutPadding (0 ms)
[ RUN      ] ONNX.ConvWithAutopadSameLower
[       OK ] ONNX.ConvWithAutopadSameLower (0 ms)
[ RUN      ] ONNX.ConvWithStridesPadding
[       OK ] ONNX.ConvWithStridesPadding (0 ms)
[ RUN      ] ONNX.ConvWithStridesNoPadding
[       OK ] ONNX.ConvWithStridesNoPadding (0 ms)
[ RUN      ] ONNX.MaxPool1d
[       OK ] ONNX.MaxPool1d (0 ms)
[ RUN      ] ONNX.MaxPool2d
[       OK ] ONNX.MaxPool2d (0 ms)
[ RUN      ] ONNX.MaxPool3d
[       OK ] ONNX.MaxPool3d (0 ms)
[ RUN      ] ONNX.AvgPool
[       OK ] ONNX.AvgPool (0 ms)
[ RUN      ] ONNX.Pow
[       OK ] ONNX.Pow (0 ms)
[ RUN      ] ONNX.Pow_broadcast
[       OK ] ONNX.Pow_broadcast (0 ms)
[ RUN      ] ONNX.ReduceMean
[       OK ] ONNX.ReduceMean (0 ms)
[ RUN      ] ONNX.ReduceProd
[       OK ] ONNX.ReduceProd (0 ms)
[ RUN      ] ONNX.ReduceSum
[       OK ] ONNX.ReduceSum (0 ms)
[ RUN      ] ONNX.ReduceSumSquare
[       OK ] ONNX.ReduceSumSquare (0 ms)
[ RUN      ] ONNX.Max
[       OK ] ONNX.Max (0 ms)
[ RUN      ] ONNX.MaxMultidirectionalBroadcast
[       OK ] ONNX.MaxMultidirectionalBroadcast (0 ms)
[ RUN      ] ONNX.MinMultidirectionalBroadcast
[       OK ] ONNX.MinMultidirectionalBroadcast (0 ms)
[ RUN      ] ONNX.MeanMultidirectionalBroadcast
[       OK ] ONNX.MeanMultidirectionalBroadcast (0 ms)
[ RUN      ] ONNX.SumMultidirectionalBroadcast
[       OK ] ONNX.SumMultidirectionalBroadcast (0 ms)
[ RUN      ] ONNX.Shape
[       OK ] ONNX.Shape (0 ms)
[ RUN      ] ONNX.RNNBatchwise
[       OK ] ONNX.RNNBatchwise (0 ms)
[ RUN      ] ONNX.RNNBidirectional
[       OK ] ONNX.RNNBidirectional (0 ms)
[ RUN      ] ONNX.RNNBidirectionalBatchwise
[       OK ] ONNX.RNNBidirectionalBatchwise (0 ms)
[ RUN      ] ONNX.RNNDefaults
[       OK ] ONNX.RNNDefaults (0 ms)
[ RUN      ] ONNX.RNNSeqLength
[       OK ] ONNX.RNNSeqLength (0 ms)
[ RUN      ] ONNX.RNNSequence
[       OK ] ONNX.RNNSequence (0 ms)
[ RUN      ] ONNX.RNNSequenceBatchwise
[       OK ] ONNX.RNNSequenceBatchwise (0 ms)
[ RUN      ] ONNX.LSTMBatchwise
[       OK ] ONNX.LSTMBatchwise (0 ms)
[ RUN      ] ONNX.LSTMBidirectional
[       OK ] ONNX.LSTMBidirectional (0 ms)
[ RUN      ] ONNX.LSTMDefaults
[       OK ] ONNX.LSTMDefaults (0 ms)
[ RUN      ] ONNX.LSTMInitialBias
[       OK ] ONNX.LSTMInitialBias (0 ms)
[ RUN      ] ONNX.LSTMPeepholes
[       OK ] ONNX.LSTMPeepholes (0 ms)
[ RUN      ] ONNX.GRUBatchwise
[       OK ] ONNX.GRUBatchwise (0 ms)
[ RUN      ] ONNX.GRUBidirectional
[       OK ] ONNX.GRUBidirectional (0 ms)
[ RUN      ] ONNX.GRUDefaults
[       OK ] ONNX.GRUDefaults (0 ms)
[ RUN      ] ONNX.GRUInitialBias
[       OK ] ONNX.GRUInitialBias (0 ms)
[ RUN      ] ONNX.GRUSeqLength
[       OK ] ONNX.GRUSeqLength (0 ms)
[ RUN      ] ONNX.Softmax1d
[       OK ] ONNX.Softmax1d (0 ms)
[ RUN      ] ONNX.Softmax2d
[       OK ] ONNX.Softmax2d (0 ms)
[ RUN      ] ONNX.Softmax3d
[       OK ] ONNX.Softmax3d (0 ms)
[ RUN      ] ONNX.Softmax4d
[       OK ] ONNX.Softmax4d (0 ms)
[ RUN      ] ONNX.ConvTranspose1d
[       OK ] ONNX.ConvTranspose1d (0 ms)
[ RUN      ] ONNX.ConvTranspose2d
[       OK ] ONNX.ConvTranspose2d (0 ms)
[ RUN      ] ONNX.ConvTransposeBias2d
[       OK ] ONNX.ConvTransposeBias2d (0 ms)
[ RUN      ] ONNX.ConvTransposeBias2dBatched
[       OK ] ONNX.ConvTransposeBias2dBatched (0 ms)
[ RUN      ] ONNX.Sqrt
[       OK ] ONNX.Sqrt (0 ms)
[ RUN      ] ONNX.Reciprocal
[       OK ] ONNX.Reciprocal (0 ms)
[ RUN      ] ONNX.Exp
[       OK ] ONNX.Exp (0 ms)
[ RUN      ] ONNX.AddBroadcast1
[       OK ] ONNX.AddBroadcast1 (0 ms)
[ RUN      ] ONNX.AddBroadcast2
[       OK ] ONNX.AddBroadcast2 (0 ms)
[ RUN      ] ONNX.AddBroadcast3
[       OK ] ONNX.AddBroadcast3 (0 ms)
[ RUN      ] ONNX.AddBroadcast4
[       OK ] ONNX.AddBroadcast4 (0 ms)
[ RUN      ] ONNX.AddBroadcast5
[       OK ] ONNX.AddBroadcast5 (0 ms)
[ RUN      ] ONNX.AddBroadcast6
[       OK ] ONNX.AddBroadcast6 (0 ms)
[ RUN      ] ONNX.AddBroadcast7
[       OK ] ONNX.AddBroadcast7 (0 ms)
[ RUN      ] ONNX.Concat0D
[       OK ] ONNX.Concat0D (0 ms)
[ RUN      ] ONNX.LayerNormalization2d
[       OK ] ONNX.LayerNormalization2d (0 ms)
[ RUN      ] ONNX.LayerNormalization4d
[       OK ] ONNX.LayerNormalization4d (0 ms)
[ RUN      ] ONNX.Equal
[       OK ] ONNX.Equal (0 ms)
[ RUN      ] ONNX.LessOrEqual
[       OK ] ONNX.LessOrEqual (0 ms)
[ RUN      ] ONNX.GreaterOrEqual
[       OK ] ONNX.GreaterOrEqual (0 ms)
[ RUN      ] ONNX.Greater
[       OK ] ONNX.Greater (0 ms)
[ RUN      ] ONNX.Less
[       OK ] ONNX.Less (0 ms)
[ RUN      ] ONNX.ExpandSameSize
[       OK ] ONNX.ExpandSameSize (0 ms)
[ RUN      ] ONNX.ExpandDiffSize
[       OK ] ONNX.ExpandDiffSize (0 ms)
[ RUN      ] ONNX.GatherAxis0
[       OK ] ONNX.GatherAxis0 (0 ms)
[ RUN      ] ONNX.GatherAxis1
[       OK ] ONNX.GatherAxis1 (0 ms)
[ RUN      ] ONNX.GatherAxis2
[       OK ] ONNX.GatherAxis2 (0 ms)
[ RUN      ] ONNX.GatherAxis3
[       OK ] ONNX.GatherAxis3 (0 ms)
[ RUN      ] ONNX.Gather2d
[       OK ] ONNX.Gather2d (0 ms)
[ RUN      ] ONNX.GatherNegativeIndices
[       OK ] ONNX.GatherNegativeIndices (0 ms)
[ RUN      ] ONNX.Slice
[       OK ] ONNX.Slice (0 ms)
[ RUN      ] ONNX.Slice_Default_Axis
[       OK ] ONNX.Slice_Default_Axis (0 ms)
[ RUN      ] ONNX.Slice_Default_Steps
[       OK ] ONNX.Slice_Default_Steps (0 ms)
[ RUN      ] ONNX.Slice_Neg
[       OK ] ONNX.Slice_Neg (0 ms)
[ RUN      ] ONNX.RangeFloat
[       OK ] ONNX.RangeFloat (0 ms)
[ RUN      ] ONNX.RangeInt
[       OK ] ONNX.RangeInt (0 ms)
[ RUN      ] ONNX.Tile5D
/builddir/build/BUILD/root-6.34.00-build/root-6.34.00/core/testsupport/src/TestSupport.cxx:79: Failure
Failed
Received unexpected diagnostic of severity 4000 at 'TUnixSystem::DispatchSignals' reading 'segmentation violation'.
Suppress those using ROOT/TestSupport.hxx
===========================================================
There was a crash (kSigSegmentationViolation).
This is the entire stack trace of all threads:
===========================================================
Thread 12 (Thread 0xe73b8b40 (LWP 143632) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 11 (Thread 0xe6bb7b40 (LWP 143633) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 10 (Thread 0xe63b6b40 (LWP 143634) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 9 (Thread 0xe5bb5b40 (LWP 143635) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 8 (Thread 0xe53b4b40 (LWP 143636) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 7 (Thread 0xe4bb3b40 (LWP 143637) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 6 (Thread 0xe43b2b40 (LWP 143638) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 5 (Thread 0xe3bb1b40 (LWP 143639) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 4 (Thread 0xe33b0b40 (LWP 143640) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 3 (Thread 0xe2bafb40 (LWP 143641) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 2 (Thread 0xe23aeb40 (LWP 143642) "TestCustomModel"):
#0  0xf490d1a7 in gomp_barrier_wait_end () from /lib/libgomp.so.1
#1  0xf490d278 in gomp_barrier_wait () from /lib/libgomp.so.1
#2  0xf490a14b in gomp_thread_start () from /lib/libgomp.so.1
#3  0xf7103b33 in start_thread () from /lib/libc.so.6
#4  0xf718adf8 in __clone3 () from /lib/libc.so.6
Thread 1 (Thread 0xf6738280 (LWP 143626) "TestCustomModel"):
#0  0xf7f1b589 in __kernel_vsyscall ()
#1  0xf716d388 in wait4 () from /lib/libc.so.6
#2  0xf716d556 in waitpid () from /lib/libc.so.6
#3  0xf70c3b18 in do_system () from /lib/libc.so.6
#4  0xf77b041c in TUnixSystem::Exec (this=0x57f675b0, shellcmd=<optimized out>) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/core/unix/src/TUnixSystem.cxx:2155
#5  TUnixSystem::StackTrace (this=<optimized out>) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/core/unix/src/TUnixSystem.cxx:2448
#6  0xf77b37ac in TUnixSystem::DispatchSignals (this=0x57f675b0, sig=kSigSegmentationViolation) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/core/unix/src/TUnixSystem.cxx:3668
#7  0xf77b3828 in SigHandler (sig=kSigSegmentationViolation) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/core/unix/src/TUnixSystem.cxx:410
#8  <signal handler called>
#9  0xf71199a6 in memcpy () from /lib/libc.so.6
#10 0x00000001 in ?? ()
#11 0x56659a6f in std::__copy_move<false, true, std::random_access_iterator_tag>::__copy_m<float const, float> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:452
#12 std::__copy_move_a2<false, float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:521
#13 std::__copy_move_a1<false, float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:548
#14 std::__copy_move_a<false, float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:555
#15 std::copy<float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:651
#16 TMVA_SOFIE_Tile5D::Session::infer (this=0xffd2fb70, tensor_input=0x57ff93d0) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/redhat-linux-build/tmva/sofie/test/Tile5D_FromONNX.hxx:37
#17 ONNX_Tile5D_Test::TestBody (this=<optimized out>) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/tmva/sofie/test/TestCustomModelsFromONNX.cxx:2889
#18 0xf7a77d14 in void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) [clone .constprop.0] () from /lib/libgtest.so.1.14.0
#19 0xf7a62956 in testing::Test::Run() () from /lib/libgtest.so.1.14.0
#20 0xf7a62b7a in testing::TestInfo::Run() () from /lib/libgtest.so.1.14.0
#21 0xf7a62dd7 in testing::TestSuite::Run() () from /lib/libgtest.so.1.14.0
#22 0xf7a6f49b in testing::internal::UnitTestImpl::RunAllTests() () from /lib/libgtest.so.1.14.0
#23 0xf7a6fadb in testing::UnitTest::Run() () from /lib/libgtest.so.1.14.0
#24 0xf7a9b0d3 in main () from /lib/libgtest_main.so.1.14.0
#25 0xf7094d43 in __libc_start_call_main () from /lib/libc.so.6
#26 0xf7094e08 in __libc_start_main_impl () from /lib/libc.so.6
#27 0x56611987 in _start ()
===========================================================
The lines below might hint at the cause of the crash. If you see question
marks as part of the stack trace, try to recompile with debugging information
enabled and export CLING_DEBUG=1 environment variable before running.
You may get help by asking at the ROOT forum https://root.cern/forum
preferably using the command (.forum bug) in the ROOT prompt.
Only if you are really convinced it is a bug in ROOT then please submit a
report at https://root.cern/bugs or (preferably) using the command (.gh bug) in
the ROOT prompt. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#9  0xf71199a6 in memcpy () from /lib/libc.so.6
#10 0x00000001 in ?? ()
#11 0x56659a6f in std::__copy_move<false, true, std::random_access_iterator_tag>::__copy_m<float const, float> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:452
#12 std::__copy_move_a2<false, float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:521
#13 std::__copy_move_a1<false, float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:548
#14 std::__copy_move_a<false, float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:555
#15 std::copy<float*, float*> (__first=<optimized out>, __last=<optimized out>, __result=<optimized out>) at /usr/include/c++/14/bits/stl_algobase.h:651
#16 TMVA_SOFIE_Tile5D::Session::infer (this=0xffd2fb70, tensor_input=0x57ff93d0) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/redhat-linux-build/tmva/sofie/test/Tile5D_FromONNX.hxx:37
#17 ONNX_Tile5D_Test::TestBody (this=<optimized out>) at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/tmva/sofie/test/TestCustomModelsFromONNX.cxx:2889
#18 0xf7a77d14 in void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) [clone .constprop.0] () from /lib/libgtest.so.1.14.0
#19 0xf7a62956 in testing::Test::Run() () from /lib/libgtest.so.1.14.0
#20 0xf7a62b7a in testing::TestInfo::Run() () from /lib/libgtest.so.1.14.0
#21 0xf7a62dd7 in testing::TestSuite::Run() () from /lib/libgtest.so.1.14.0
#22 0xf7a6f49b in testing::internal::UnitTestImpl::RunAllTests() () from /lib/libgtest.so.1.14.0
#23 0xf7a6fadb in testing::UnitTest::Run() () from /lib/libgtest.so.1.14.0
#24 0xf7a9b0d3 in main () from /lib/libgtest_main.so.1.14.0
#25 0xf7094d43 in __libc_start_call_main () from /lib/libc.so.6
#26 0xf7094e08 in __libc_start_main_impl () from /lib/libc.so.6
#27 0x56611987 in _start ()
===========================================================
CMake Error at /builddir/build/BUILD/root-6.34.00-build/root-6.34.00/cmake/modules/RootTestDriver.cmake:232 (message):
  error code: 139
```
The autogenerated file Tile5D_FromONNX.hxx: 
```
//Code generated automatically by TMVA for Inference of Model file [Tile5D.onnx] at [Fri Dec 13 09:18:20 2024] 

#ifndef ROOT_TMVA_SOFIE_TILE5D
#define ROOT_TMVA_SOFIE_TILE5D

#include <vector>
#include "TMVA/SOFIE_common.hxx"

namespace TMVA_SOFIE_Tile5D{
struct Session {
// initialized tensors
int64_t tensor_repeats[5] = {2, 1, 2, 1, 3};

//--- declare and allocate the intermediate tensors
std::vector<float> fTensor_output = std::vector<float>(0);
float * tensor_output = fTensor_output.data();


Session(std::string = "") {
//---- allocate the intermediate dynamic tensors
}

std::vector<float> infer(float* tensor_input){
///-------- Tile operator
{
std::vector<int> input_shape = { 2 , 2 , 2 , 3 , 3 };
std::vector<int> repeats = { 2 , 0 , 1 , 0 , 2 };
int inputLength = 72;
int s = 1;
for (int i = 4; i >=0; i--) {
   int r = repeats[i];
   int i_offset = 0, o_offset = 0;
   s = s * input_shape[i];
   if (i == 4) {
      for (int j = 0; j < inputLength/s ; j++) {
         for (int k = 0; k < r ; k++) {
            std::copy(tensor_input+ i_offset, tensor_input+ i_offset + s, tensor_output+ o_offset);
            o_offset += s;
         }
         i_offset += s;
      }
   } else {
      for (int j = inputLength/s - 1 ; j>=0; j--) {
         o_offset = j*s*r;
         i_offset = j*s;
         for (int k = 0; k < r ; k++) {
            std::copy(tensor_output+ i_offset, tensor_output+ i_offset + s, tensor_output+ o_offset);
            o_offset += s;
         }
      }
   }
   s *= r;
   inputLength *= r;
}
}
   return fTensor_output;
}
};   // end of Session
} //TMVA_SOFIE_Tile5D

#endif  // ROOT_TMVA_SOFIE_TILE5D
```
In this file the output tensor is declared with length 0:
```
std::vector<float> fTensor_output = std::vector<float>(0);
```
But then continues to copy data into this array,

### Reproducer

I get a segfault if I run the test on ix86. On other architectures usually not.

### ROOT version

6.34.00, 6.34.02

### Installation method

Build from source

### Operating system

Fedora 41 i686, Fedora rawhide i686.

### Additional context

_No response_

--- a/tmva/sofie/inc/TMVA/ROperator_Tile.hxx
+++ b/tmva/sofie/inc/TMVA/ROperator_Tile.hxx
@@ -51,12 +51,17 @@ public:
       }
       fShapeInput=model.GetTensorShape(fNInput);
 
-         // Retrieve the data pointer for the repeats tensor
+      // if repeats vector is not initialized we cannot deduce shape of output
+      // not support for time being this case
+      if (!model.IsInitializedTensor(fNRepeats)) {
+         throw std::runtime_error("TMVA SOFIE Tile Op: non-initialized repeats input is not supported");
+      }
+
+      // Retrieve the data pointer for the repeats tensor
       auto repptr = model.GetInitializedTensorData(fNRepeats);
       // Cast the raw pointer to the appropriate type (size_t*)
-      auto repeat_shape = static_cast<size_t*>(repptr.get());
-
-      if (repeat_shape == nullptr) {
+      auto repeats_data = static_cast<int64_t*>(repptr.get());
+      if (repeats_data == nullptr) {
         throw std::runtime_error("Failed to retrieve the data for the repeats tensor.");
       }
       // Get the shape of the repeats tensor to determine the number of elements
@@ -66,12 +71,18 @@ public:
          throw std::runtime_error("Repeats tensor is not 1D.");
       }
       size_t num_elements = repeats_shape[0];
-      // Convert the data to a vector
-      std::vector<size_t> repeats_vector(repeat_shape, repeat_shape + num_elements);
+      // Convert the data to a vector of size_t
+      std::vector<size_t> repeats_vector(num_elements);
+      std::copy(repeats_data, repeats_data + num_elements, repeats_vector.begin());
+
 
       fShapeY = ShapeInference({fShapeInput,repeats_vector})[0];
 
       model.AddIntermediateTensor(fNY, model.GetTensorType(fNInput), fShapeY);
+
+      if (model.Verbose())
+         std::cout <<  "Tile: " << fNInput << " " << ConvertShapeToString(fShapeInput) << " -> " << fNY << " with shape " << ConvertShapeToString(fShapeY)
+            << " given repeats " << ConvertShapeToString(repeats_vector) << std::endl;
    }
 
    std::string Generate(std::string OpName){
@@ -89,17 +100,13 @@ public:
       std::string output = "tensor_" + fNY;
       out << "///-------- Tile operator\n";
       out << "{\n"; // add scope to re-use same names
-      out << "std::vector<int> input_shape = " << ConvertShapeToString(fShapeInput) << ";\n";
-      std::vector<size_t> repeats = fShapeY;
-      for (size_t i = 0; i < repeats.size(); i++)
-         repeats[i] /= fShapeInput[i];
+      out << "const int input_shape[" << fShapeInput.size() << "] = " << ConvertShapeToString(fShapeInput) << ";\n";
 
-      out << "std::vector<int> repeats = " << ConvertShapeToString(repeats) << ";\n";
       out << "int inputLength = " << ConvertShapeToLength(fShapeInput) << ";\n";
       out << "int s = 1;\n";
       // loop from inverse dim order
       out << "for (int i = " << fShapeInput.size()-1 << "; i >=0; i--) {\n";
-      out << SP << "int r = repeats[i];\n";
+      out << SP << "int r = tensor_" << fNRepeats << "[i];\n";
       // we cannot exclude case where repeats=1 since we need offset
       //out << SP << "if (r == 1 && i < " << fShapeInput.size()-1 <<  ") continue;\n";
       out << SP << "int i_offset = 0, o_offset = 0;\n";

